src/api/libcellml/component.h

```diff
@@ -416,21 +416,20 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
      */
     ComponentPtr clone() const;
 
+private:
+    Component(); /**< Constructor @private*/
+    explicit Component(const std::string &name); /**< Constructor named @private */
+
+    bool doAddComponent(const ComponentPtr &component) override; /**< Virtual method for implementing addComponent, @private */
+
     /**
      * @brief Set the import source of this component.
      *
-     * If this component is already located in a Model instance, then the
-     * import source is added to the Model too.
+     * Virtual method implementing ImportedEntity::setImportSource, @private.
      *
      * @param importSource The @c ImportSourcePtr to add to this @c Component.
      */
-    void setImportSource(ImportSourcePtr &importSource);
-
-private:
-    Component(); /**< Constructor @private*/
-    explicit Component(const std::string &name); /**< Constructor named @private */
-
-    bool doAddComponent(const ComponentPtr &component) override; /**< Virtual method for implementing addComponent, @private */
+    void doSetImportSource(const ImportSourcePtr &importSource) override;
```

Comment:
Just as a reminder, `doAddComponent()` and `doSetImportSource()` will have to be removed from the API at some point. They really have nothing to do here.